app/src/androidTest/java/com/github/h3lp3rs/h3lp/HelperPageActivityTest.kt

```diff
@@ -154,6 +154,24 @@ class HelpPageActivityTest : H3lpAppTest() {
         }
     }
 
+    @Test
+    fun showsPopUpWhenNotSignedIn(){
+        // Not signed in
+        userUid = null
+
+        launchAndDo {
+
+            // We can close the popup => It's displayed :)
```

Comment:
You could use the isDisplayed() method! but we trust you 😉

Reply:
Actually not really. I cannot use the isDisplayed on the popup layout (doesn't work) and calling isDisplayed() on the button only is useless since it's a requirement to clicking on it